src/BuiltInTools/dotnet-watch/DotNetWatcher.cs

```diff
@@ -85,10 +85,8 @@ public async Task WatchAsync(ProcessSpec processSpec, CancellationToken cancella
                 using var combinedCancellationSource = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken, currentRunCancellationSource.Token);
                 using var fileSetWatcher = new FileSetWatcher(fileSet, context.Reporter);
 
-                context.Reporter.Verbose($"Running {processSpec.ShortDisplayName()} with the following arguments: '{processSpec.GetArgumentsDisplay()}'");
                 var processTask = _processRunner.RunAsync(processSpec, combinedCancellationSource.Token);
-
-                context.Reporter.Output("Started", emoji: "🚀");
+                context.Reporter.Output("Started");
```

Comment:
The `WatchStartedMessage` seems to have ⌚in it, do we need to add the emoji to the Started output?

Reply:
⌚ is the default value of the optional parameter.